CondFormats/CTPPSReadoutObjects/interface/CTPPSPixelROC.h

```diff
@@ -0,0 +1,112 @@
+#ifndef CTPPSReadoutObjects_PixelROC_H
+#define CTPPSReadoutObjects_PixelROC_H
+
+#include "CondFormats/Serialization/interface/Serializable.h"
+
+#include "CondFormats/CTPPSReadoutObjects/interface/CTPPSPixelIndices.h"
+
+#include <boost/cstdint.hpp>
+#include <string>
+
+/** \class PixelROC
+ * Represents ReadOut Chip of DetUnit. 
+ * Converts pixel coordinates from Local (in ROC) to Global (in DetUnit).
+ * The Local coordinates are double column (dcol) and pixel index in dcol.
+ * The Global coordinates are row and column in DetUnit.
+ */
+
+
+class CTPPSPixelROC {
+public:
+
+CTPPSPixelROC() : theDetUnit(0), theIdDU(0), theIdLk(0) {
+    theIndices = new CTPPSPixelIndices();
```

Comment:
I don't think it's strictly necessary, just like it. If you think it's worth I can try not to use a pointer...